postgres/metadata.csv

```diff
@@ -98,4 +98,9 @@ postgresql.slru.truncates,count,,,,"Number of truncates for this SLRU (simple le
 postgresql.transactions.duration.max,gauge,,nanosecond,,"The age of the longest running transaction per user, db and app. (DBM only)",0,postgres,postgres transactions duration max,
 postgresql.transactions.duration.sum,gauge,,nanosecond,,"The sum of the age of all running transactions per user, db and app. (DBM only)",0,postgres,postgres transactions duration sum,
 postgresql.wal_age,gauge,,second,,The age in seconds of the oldest WAL file.,0,postgres,wal age,
+postgresql.wal_receiver.count,gauge,,,,"Count of WAL receiver, it can be 0 or 1. If it's 1, this means that the replica is connected to the primary's WAL sender. If it's 0, either it's primary or it's a replica disconnected from the primary.",0,postgres,wal receiver count,
```

Comment:
Is this a count or is this a boolean flag? It sounds like a boolean flag (`The pg_stat_wal_receiver view will contain only one row`). I would suggest an alternative API for these metrics:

If receiver:
* type: `gauge`
* value: `1`
* tag: `wal_receiver_status:connected`

If not receiver:
* type: `gauge`
* value: `1`
* tag: `wal_receiver_status:disconnected`

This will allow you to plot: `postgresql.wal_receiver.connected{*} by {wal_receiver_status}` on a graph.

Reply:
Yeah, I was not super happy with this metric. 
I've tried replacing with your suggestion with one small different: I'm reusing the `status` tag and set it to `disconnected` when there's no WAL receiver.